src/typeDefs/types.ts

```diff
@@ -385,13 +385,23 @@ export const types = gql`
       before: String
       first: PositiveInt
       last: PositiveInt
-    ): UserTagsConnection
+    ): UserTagsConnectionResult
     usersAssignedTo(
       after: String
       before: String
       first: PositiveInt
       last: PositiveInt
-    ): UsersConnection
+    ): UsersConnectionResult
+  }
+
+  type UsersConnectionResult {
+    connectionData: UsersConnection
+    connectionErrors: [PaginationError]
+  }
+
+  type UserTagsConnectionResult {
+    connectionData: UserTagsConnection
+    connectionErrors: [PaginationError]
```

Comment:
`[ConnectionError!]!`, make it a non null list of non null pagination errors, if no errors are found it'll be an empty list